osh.spec

```diff
@@ -75,7 +77,8 @@ Recommends: osh-worker-conf
 Obsoletes: covscan-worker < %{version}
 
 %description worker
-OpenScanHub worker
+OpenScanHub worker.
+It is used to perform scans.
```

Comment:
I am not sure if this is an improvement really.  I would say that all the subpackages are used "to perform scans".  I would reuse the description we already have in `kobo`: https://src.fedoraproject.org/rpms/kobo/blob/3b18f70fbe4809a4aa7be89b6c93b3d716738b60/f/kobo.spec#_73